src/Symfony/Component/Notifier/Bridge/Slack/SlackOptions.php

```diff
@@ -58,18 +58,6 @@ public function getRecipientId(): ?string
         return $this->options['recipient_id'] ?? null;
     }
 
-    /**
-     * @return $this
-     *
-     * @deprecated since Symfony 5.1, use recipient() instead.
-     */
-    public function channel(string $channel): self
-    {
-        trigger_deprecation('symfony/slack-notifier', '5.1', 'The "%s()" method is deprecated, use "recipient()" instead.', __METHOD__);
```

Comment:
Please remove `symfony/deprecation-contracts` from the composer.json file of the bridge.